src/tools/rustdoc-js/tester.js

```diff
@@ -423,11 +423,7 @@ function parseOptions(args) {
     };
 
     for (var i = 0; i < args.length; ++i) {
-        if (args[i] === "--resource-suffix"
-            || args[i] === "--doc-folder"
-            || args[i] === "--test-folder"
-            || args[i] === "--test-file"
-            || args[i] === "--crate-name") {
+        if (correspondances[args[i]] !== undefined) {
```

Comment:
```suggestion
        if (Object.hasOwn(correspondances, args[i])) {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
Also, a nit: "correspondances" should be spelled "correspondences". While you're at it, do you want to fix the spelling?

Reply:
Sure!